.changeset/eighty-comics-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+Translated Vietnamese UI.
```

Comment:
Tiny detail, but se’ve been using this format for changelogs recently, so would be nice to be consistent:

```suggestion
Updates Vietnamese UI translations
```